src/containers/edm/propertytypes/PropertyTypeDetailsContainer.js

```diff
@@ -103,6 +127,19 @@ class PropertyTypeDetailsContainer extends React.Component<Props> {
   }
 }
 
+function mapStateToProps(state :Map<*, *>, ownProps) :Object {
+  const propertyTypeId = ownProps.propertyType.get('id');
+  const entityTypes :OrderedSet<string> = state.getIn(['edm', 'entityTypes', 'entityTypes'], List())
```

Comment:
You don't need to call `.toOrderedSet()` nor `.toList()`, just `.filter()`. Also, use `:List<string>` instead.